### ISSUE TITLE
ntop.conf not generated/doesn't work

### ISSUE THREAD

**Issue by hikigaya5** [gsass1/NTop#71]
I was searching for the ntop.conf file to change the color scheme but I didn't find it anywhere. I created my own ntop.conf file and located it in the same folder as the executable as I saw in another issue but this doesn't seem to work when I try to change colors.
Also the link in the ntop.conf example file of the readme doesn't direct you to the page of the possible colors, it redirects you to other thing.

**Comment by Demitrix2k** [comment]
I tried to do the same using the template and modifying it, placing it alongside ntop.exe in the folder structure for 

`AppData\Local\Microsoft\WinGet\Packages\gsass1.NTop_Microsoft.Winget.Source_8wekyb3d8bbwe`

tried if i could set `Env:NTOP_CONFIG_HOME=`  no such luck either.

I see people back to 2021 with questions about this not answered, would appreciate one and possibly updating main page regarding this.

**Comment by hikigaya5** [comment]
Yes I tried. Also where are the possible colors? Because the link in the generated ntop.conf file redirects you to a CHAR_INFO structure that says it has only 3 possible colors (red, green and blue) and it doesn't seem quite clear to me.

**Comment by gsass1** [comment]
@hikigaya5 The way it's currently implemented is not very user friendly. You have to currently manually create the RGB value, I think there's a tool for this but I'm not sure

**Comment by gsass1** [comment]
In the newest commit the config file is read relative to the executable file (the intended behaviour, before it was working directory).

I tested this and it should work, let me know!

**Comment by Demitrix2k** [comment]
Can confirm, now works with config file within the same folder as the executable.
